api/routes/index.js

```diff
@@ -1,4 +1,27 @@
-import express from 'express';
-const router = express.Router();
+export function register(server, options, next) {
+  /* Serve static files from public folder */
+  server.route({
```

Comment:
This should probably also be in it's own module, we are going for endpoint style design.